src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked.
```

Comment:
I _think_ we actually prefer these right above the class declaration

Reply:
+1 on having just imports at the very top. Not sure there's any specific reason. Consistency?